data/Arts2023Longitudinal.yaml

```diff
@@ -0,0 +1,3857 @@
+# yaml-language-server: $schema=.schema.yaml
+title: Longitudinal and quantitative fecal shedding dynamics of SARS-CoV-2, pepper
+  mild mottle virus, and crAssphage
+doi: 10.1128/msphere.00132-23
+description: >
+  The authors present longitudinal, quantitative fecal shedding data for SARS-CoV-2
+  RNA, pepper mild mottle virus (PMMoV) RNA, and crAss-like phage (crAssphage) DNA
+  from 48 COVID-19 patients. Abundances were quantified using (RT)-ddPCR assays targeting
+  the N and ORF1a genes. The data were obtained from supplementary material.
+analytes:
+  stool_SARSCoV2_N:
+    description: >
+      Concentration of RNA of the N gene quantified using (RT)-ddPCR in stool samples.
+      The concentration was quantified in gene copies per dry weight of stool. The
+      limit of blank (LOB), determined as the upper 95% confidence limit of the negative
+      extraction control, ranged from 11.2 to 1,550 gc/mg-dry weight.
+    limit_of_quantification: unknown
+    limit_of_detection: unknown
+    limit_of_blank: 1550
+    specimen: stool
+    biomarker: SARS-CoV-2
+    gene_target: N
+    unit: gc/dry gram
+    reference_event: symptom onset
+  stool_SARSCoV2_ORF1a:
+    description: >
+      Concentration of RNA of the ORF1a gene quantified using (RT)-ddPCR in stool
+      samples. The concentration was quantified in gene copies per dry weight of
+      stool. The limit of blank (LOB), determined as the upper 95% confidence limit
+      of the negative extraction control, ranged from 11.2 to 1,550 gc/mg-dry
+      weight.
+    limit_of_quantification: unknown
+    limit_of_detection: unknown
+    limit_of_blank: 1550
+    specimen: stool
+    biomarker: SARS-CoV-2
+    gene_target: ORF1a
+    unit: gc/dry gram
+    reference_event: symptom onset
+  stool_PMMoV:
+    description: >
+      Concentration of PMMoV RNA was measured by ddPCR on the same day as the SARS-CoV-2.
+      The concentration was quantified in gene copies per dry weight of stool.
+    limit_of_quantification: unknown
+    limit_of_detection: unknown
+    limit_of_blank: 400
+    specimen: stool
+    biomarker: PMMoV
+    unit: gc/dry gram
+    reference_event: symptom onset
+  stool_crAssphage:
+    description: >
+      Concentration of crAssphage DNA was measured using qPCR. The reported number
+      is either the measured concentration of crAssphage or the Limit of Quantification
+      (LOQ) if the concentration wasn't quantifiable.
+    limit_of_quantification: unknown
+    limit_of_detection: 25
+    specimen: stool
+    biomarker: crAssphage
+    unit: gc/dry gram
+    reference_event: symptom onset
+participants:
+- attributes:
+    age: 39
```

Comment:
Hi @tillahoffmann @WenfeiXiao , in Liu2024Longitudinal.yaml, I used float64 format for `age` in Python. So I will have 39.0 instead of 39 for this case. I did this based on previous discussions for potential data from young children with `age` as year and month format. I do not think this will fail the schema but I just want to bring this up to your attention.

Reply:
Great, thanks for raising this! As you mentioned, either integers or floats should work ok here.